src/class/eventManagement/dispatcher.class.ts

```diff
@@ -503,11 +503,15 @@ export class Dispatcher<T extends GenericEvent = GenericEvent> extends EventEmit
       }
 
       await Promise.all(toHandle);
+
+      this.logger.info(`[${inactiveIncomers.map(
+        (incomer) => `(name:${incomer.name}|uuid:${incomer.providedUUID ?? incomer.baseUUID})`
+      ).join(",")}], Removed Incomer`);
```

Comment:
Ca ne risque pas de faire un log à rallonge ? Un `join("\n")` ne serait pas mieux ?

Reply:
Line break not working cuz of use of `pino-pretty`

![image](https://github.com/MyUnisoft/events/assets/39910164/28ef243e-6444-4aa0-ac1e-beff1309a27f)